guidance.md

```diff
@@ -262,5 +244,6 @@ Example response when criteria do apply:
 Extra notes
 
 - Which AUCs do we need to support? Do they define different scoring systems? Do we need common roll-up codes like "Yes, Appropriate", "No, inappropriate" or "Indeterminate" ? Can we provide extensibility for AUCs to express their own scores too, directly? Do we capture which qualified QCDM who provided the score?
+   - CMS only recognizes three values for appropriateness on the claim: appropriate, not appropriate and no criteria apply. CDSMs and providers benefit from finer-grained appropriateness scores, which are often numeric and specific to that qCDSM or AUC method or PLE. The qCDSM may enhance it's service to the provider by returning both the CMS appropriateness indiciation and a more discrete score on the param-rating extension, through the use of a translation coding. TODO - example translation coding needed!
```

Comment:
Re: CMS, we should discuss. Officially I think CMS just says that when submitting a bill, the system needs to attest that (https://www.cms.gov/Outreach-and-Education/Medicare-Learning-Network-MLN/MLNMattersArticles/Downloads/MM10481.pdf):

> Ordering Professional Consulted A Qualified Clinical Decision Support Mechanism For This Service And The Related Data Was Provided To The Furnishing Professional) is available for this reporting. The modifier may be:
> * Used when the furnishing professional is aware of the result of the ordering professional’s consultation with a CDSM for that patient,
> * Reported on the same claim line as the CPT code for an advanced diagnostic imaging service furnished in an applicable setting and paid for under an applicable payment system, and,
> * Reported on both the facility and professional claim. 

In terms of specific capabilities, I see (https://www.govinfo.gov/content/pkg/FR-2016-11-15/pdf/2016-26668.pdf):

> All CDSMs must meet the requirements under section 1834(q)(3)(B) of the Act, which specifies that a mechanism must: Make available to the ordering professional applicable AUC and the documentation supporting the appropriateness of the applicable imaging service that is ordered; where there is more than one applicable appropriate use criterion specified for an applicable imaging service, indicate the criteria it uses for the service; determine the extent to which an applicable imaging service that is ordered is consistent with the applicable AUC; generate and provide to the ordering professional documentation to demonstrate that the qualified CDSM was consulted by the ordering professional; be updated on a timely basis to reflect revisions to the specification of applicable AUC; meet applicable privacy and security standards; and perform such other functions as specified by the Secretary (which may include a requirement to provide aggregate feedback to the ordering professional).